File: riva/client/package_info.py
# SPDX-FileCopyrightText: Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
# SPDX-License-Identifier: MIT

MAJOR = 2
MINOR = 9
PATCH = 0
PRE_RELEASE = 'rc0'

# Use the following formatting: (major, minor, patch, pre-release)
VERSION = (MAJOR, MINOR, PATCH)

__shortversion__ = '.'.join(map(str, VERSION[:3]))
__version__ = '.'.join(map(str, VERSION[:3])) + ''.join(VERSION[3:])

__package_name__ = 'nvidia-riva-client'
__contact_names__ = 'Anton Peganov'
__contact_emails__ = 'apeganov@nvidia.com'
__homepage__ = 'https://github.com/nvidia-riva/python-clients'
__repository_url__ = 'https://github.com/nvidia-riva/python-clients'
__download_url__ = 'hhttps://github.com/nvidia-riva/python-clients/releases'
__description__ = "Python implementation of the Riva Client API"
__license__ = 'MIT'
__keywords__ = 'deep learning, machine learning, gpu, NLP, ASR, TTS, NMT, nvidia, speech, language, Riva, client'
__riva_version__ = "2.9.0"
__riva_release__ = "23.01"
__riva_models_version__ = "2.9.0"

File: riva-ts-client/src/client/package_info.ts
export const VERSION = {
    MAJOR: 2,
    MINOR: 18,
    PATCH: 0,
    PRE_RELEASE: 'rc0'
} as const;

export const PACKAGE_INFO = {
    shortversion: `${VERSION.MAJOR}.${VERSION.MINOR}.${VERSION.PATCH}`,
    version: `${VERSION.MAJOR}.${VERSION.MINOR}.${VERSION.PATCH}${VERSION.PRE_RELEASE}`,
    packageName: 'nvidia-riva-client',
    contactNames: 'Anton Peganov',
    contactEmails: 'apeganov@nvidia.com',
    homepage: 'https://github.com/nvidia-riva/python-clients',
    repositoryUrl: 'https://github.com/nvidia-riva/python-clients',
    downloadUrl: 'https://github.com/nvidia-riva/python-clients/releases',
    description: 'TypeScript implementation of the Riva Client API',
    license: 'MIT',
    keywords: [
        'deep learning',
        'machine learning',
        'gpu',
        'NLP',
        'ASR',
        'TTS',
        'NMT',
        'nvidia',
        'speech',
        'language',
        'Riva',
        'client'
    ],
    rivaVersion: '2.18.0',
    rivaRelease: '24.12',
    rivaModelsVersion: '2.18.0'
} as const;
